### PR TITLE
Feature/lock unlock buttons

### LINKED ISSUE #6: Iteration 2, Locking/Unlocking buttons & innerHTML
- When the page is initially loaded, all colors should have an unlocked icon in the bottom right corner
- When a user clicks on the unlocked icon, a locked icon should appear; The locked/unlocked icons should toggle back and forth as the user clicks them

### PR DESCRIPTION
**Pull request by jalbe0076**
- Added comp images to asset directory. 
- Linked JS in HTML
- Added event listener looking for className on the grandparent target and changing the displayed image src when clicked
- Added styling to display lock image matching comp
- Tested by clicking locks in different orders to randomize what locks were locked and the others opened. Clicked across grandparent and parent targets to ensure the image toggle happened only when over intended target
The PR will close #6

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
 function getRandomHex() {
     return (Math.floor(Math.random() * 16777216).toString(16).padStart(6, 0));
+}
+
+function lockToggle(event) {
+    if (event.target.src === 'file:///Users/jason/turing/1mod/projects/coloRandom/assets/unlocked.png') {
+        event.target.src = 'file:///Users/jason/turing/1mod/projects/coloRandom/assets/locked.png'
+    } else {
+        event.target.src = 'file:///Users/jason/turing/1mod/projects/coloRandom/assets/unlocked.png'
+    }
```
Comment (mbenfowler) [inline main.js]: this full path for these images isn't gonna work anywhere other than on your system!  Please use similar relative paths like you used up in your html changes.

Screenshot 2023-04-12 at 10 10 04 AM

File: styles.css [hunk 2]
```diff
@@ -52,11 +52,26 @@ button {
...
+    right: 0;
+    height: 30px;
+    width: 30px;
+    padding: 8px;
+    text-align: center;
+}
+
+.lock-box:hover {
+    transform: scale(1.3);
+}
```
Comment (mbenfowler) [inline styles.css:73]: Nice!

**Review by mbenfowler — CHANGES_REQUESTED** [review]

File: main.js (hunk 1 above)
Comment (jalbe0076) [reply main.js]: Fixed relative path issue. `event.target.src` did not work with the  conditional. Had to use the `getAttribute` function instead.

**Review by mbenfowler — APPROVED** [review]

**Review by SethBurns — APPROVED** [review]
Looks solid, as we discussed, maybe changing name of function from lockToggle to toggleLock would a better naming convention.

**Pull request merged**